dto.c

```diff
@@ -13,6 +13,7 @@
 #include <sys/mman.h>
 #include <cpuid.h>
 #include <linux/idxd.h>
+#include <sys/types.h>
```

Comment:
The code compiles without this #include. Can you please check if it is really needed?